src/dnlib.csproj

```diff
@@ -15,6 +15,8 @@
     <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\dnlib.snk</AssemblyOriginatorKeyFile>
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <FrameworkPathOverride Condition="'$(OS)' != 'Windows_NT'">/usr/lib/mono/2.0-api/</FrameworkPathOverride>
```

Comment:
It shows some path that .NET Core installer will search for, so I think it is ok to hard-code these path for now. I have added a comment link to the issue.